src/installation/integrations.md

```diff
@@ -26,6 +26,14 @@ import compiledTemplate from "./template.handlebars";
 The [handlebars-webpack-plugin](https://github.com/sagold/handlebars-webpack-plugin) uses Handlebars to build your
 HTML-pages statically when compiling your application
 
+## Webpack: html-bundler-webpack-plugin
+
+The [html-bundler-webpack-plugin](https://github.com/webdiscus/html-bundler-webpack-plugin) uses various templating
+engines, [including Handlebars](https://github.com/webdiscus/html-bundler-webpack-plugin#using-the-handlebars),
+to render templates and bundle styles and scripts into generated HTML.
+
+[View an usage example](https://stackblitz.com/edit/webpack-webpack-js-org-mxbx4t?file=webpack.config.js)
```

Comment:
Can you remove this line? I think it isn't necessary, since your GitHub project already contains links to examples.

Reply:
@jaylinski the `View an usage example` line is removed.